test/ds1621-sync.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+// Test the sychronous raspio-i2c methods
+// Assumes that a DS1621 temperature sensor is wired up to the Pi
+
+var assert = require('assert'),
```

Comment:
The standard way to write unit tests is to use [NodeUnit](https://github.com/caolan/nodeunit) and [Sinon.JS](http://sinonjs.org/), not manually doing asserts. Can you re-write these?

Alternately, you can also just remove them alltogether, if you prefer. I have started some work on an automated test-suite for raspi-io that will use an Arduino Uno with some custom C code to do the testing, which will include I2C (the Pi will send the Arduino I2C commands, which the Arduino will record and verify, and the Arduino will also send data back to the Pi, which the Pi will record and verity).